src/AniListNet/Objects/Character/CharacterSort.cs

```diff
@@ -4,7 +4,11 @@ namespace AniListNet.Objects;
 
 public enum CharacterSort
 {
+    [EnumMember(Value = "ID")] Id,
+    [EnumMember(Value = "ID_DESC")] IdDesc,
     [EnumMember(Value = "ROLE")] Role,
+    [EnumMember(Value = "ROLE_DESC")] RoleDesc,
     [EnumMember(Value = "SEARCH_MATCH")] Relevance,
-    [EnumMember(Value = "FAVOURITES")] Favorites
+    [EnumMember(Value = "FAVOURITES")] Favorites,
+    [EnumMember(Value = "FAVOURITES_DESC")] FavoritesDesc
```

Comment:
Don't need to add `_DESC` properties because you can just set it in `SearchCharacterFilter` in at [this property](https://github.com/dentolos19/AniListNet/blob/394f31e9b872d832e9266859ffbd09c04921cf7c/src/AniListNet/Parameters/SearchCharacterFilter.cs#L11).